src/css/custom.css

```diff
@@ -151,3 +152,12 @@ html[data-theme='dark'] .faq .img {
   font-weight: 700;
   font-style: normal;
 }
+
+.lego-card {
+  box-shadow: 0 1px 2px rgba(0, 0, 0, 0.15);
+  transition: box-shadow 0.3s ease-in-out;
+}
+
+.lego-card:hover {
+  box-shadow: 0 2.5px 5px rgba(0, 0, 0, 0.8);
+}
```

Comment:
These seem to be specifically related to the component only. I would suggest putting them in `components/LegoGallery/LegoGallery.modules.scss` (as done for all the other components)